README.md

```diff
@@ -36,22 +37,23 @@ Frame is a native web3 interface that lets you sign data, securely manage accoun
   - macOS, Windows and Linux!
 
 ### Talks
-  - [Frame at Aracon](https://www.youtube.com/watch?v=wlZWLiy2GD0)
 
-### Installation
+- [Frame at Aracon](https://www.youtube.com/watch?v=wlZWLiy2GD0)
 
-#### Downloads
-  - [All Releases](https://github.com/floating/frame/releases)
+### Downloads
 
-#### Arch Linux
+- [All Releases](https://github.com/floating/frame/releases)
 
-If you use an arch-based distro, you can use an AUR Helper like [yay](https://github.com/Jguer/yay) to install Frame by running `yay -S frame-eth` or for the development version: `yay -S frame-eth-dev`.
+### Run Source
 
-#### Run Source
 ```bash
 # Clone
 › git clone https://github.com/floating/frame
 
+# Use node v16
+› nvm install 16
```

Comment:
@cantobanano Frame's current version of Electron uses Node v16.14.2, let's make this more specific to avoid any confusion or conflicts

Reply:
> I think it's still `16.13.2`? I was considering to upgrade to `16.14.x` to get the `AbortSignal.timeout()` backport but we went with a regular AbortController approach in the end because the former wasn't working well.

we're on Electron 19.0.4 right? https://www.npmjs.com/package/electron-releases

Reply:
20.1.4 now so we're both wrong.  `16.15.0`.  I was going off the build scripts, will update those.

Reply:
Yes, it makes sense to merge this after the dep PR so we are documenting the latest.